src/commands/ms/add.ts

```diff
@@ -15,8 +15,8 @@ export default class Add extends Command {
   static flags = {
     stack: Flags.string({
       description: 'Microservice stack',
-      options: [MicroServiceStack.SpringBoot, MicroServiceStack.Node],
-      default: MicroServiceStack.SpringBoot
+      options: [MicroserviceStack.SpringBoot, MicroserviceStack.Node],
```

Comment:
I would replace this with `Object.values(MicroserviceStack)`, so we are dynamic (and the same in MFE add command).